pkg/uefi/file.go

```diff
@@ -178,23 +178,24 @@ func (a *fileAttr) setLarge(large bool) {
 }
 
 // Checks if we need to checksum the file body
-func (a fileAttr) hasChecksum() bool {
+func (a fileAttr) HasChecksum() bool {
 	return a&0x40 != 0
 }
 
 // HeaderLen is a helper function to return the length of the file header
 // depending on the file size
 func (f *File) HeaderLen() uint64 {
```

Comment:
I think you can drop 'is a helper function'
HeaderLen returns the length of the file header.